css/css/content.css

```diff
@@ -0,0 +1,142 @@
+/* Main content */
+.main-content-container {
+  display: flex;
+}
+.main-content-title {
+  color: var(--tertiary-color);
+}
+.highlight {
+  color: var(--tertiary-color);
+}
+.main-content-btn {
+  padding: 0.7rem 1.1rem;
+  text-decoration: none;
+  font-size: var(--font-size-5);
+  font-weight: var(--font-weight-2);
+  color: var(--tertiary-color);
+  background-color: var(--complementary-color);
+}
+.main-content-btn:hover,
+.main-content-btn:focus {
+  box-shadow: 0px 12px 12px rgba(0, 0, 0, 0.25),
```

Comment:
can make a var for shadows if used in multiple files
`--var-name: 0px 12px 12px rgba(0, 0, 0, 0.25)`